builtins.go

```diff
@@ -681,6 +682,69 @@ func builtinMd5(i *interpreter, trace traceElement, x value) (value, error) {
 	return makeValueString(hex.EncodeToString(hash[:])), nil
 }
 
+func builtinBase64(i *interpreter, trace traceElement, input value) (value, error) {
+	var byteArr []byte
+
+	var sanityCheck = func(v int) (string, bool) {
+		if v < 0 || v > 256 {
```

Comment:
This was originally `!(v < 256 && v >= 0)`. If I understand correctly, the same logic should be `v >= 256` or `v > 255` here.

One more bike shed... 😝 I prefer the mathy/range look of less-than: `v < 0 || 255 < v`

Reply:
oh yes.. the number should be 255.

Reply:
I agree that `N < v && v < M` looks best. However, it reads differently when it's an `||` (or). And I think `v < N || v > M` feels better.

Reply:
wait.. I now see the desired range in `v < 0 || 255 < v`...